### PR TITLE
hide messages from before seed

### PR DESCRIPTION
**Pull request by Demarchist1**
Is there a way to filter at the firebase back end rather than the front end?

### CODE REVIEW

**Comment by thecadams** [comment]
You can filter in the `useFirestoreConnect` call. Maybe we should always filter venue messages older than X milliseconds.

(We should never hide old private messages of course)

**Comment by thecadams** [comment]
This PR is a solid idea, a more general solution would account for the code being used in non burn mode and for other SparkleVerse events; would you be willing to have a think about a more generalised solution? Going to proactively approve so you’re not blocked. Great stuff.

File: src/components/organisms/ChatDrawer/ChatDrawer.tsx [hunk 1]
```diff
@@ -62,6 +63,7 @@ const ChatDrawer: React.FC<PropsType> = ({
       chatContext.sendRoomChat(user.uid, roomName, data.messageToTheBand);
     reset();
   };
+  const BEFORE_SEED = SEED_START_UTC_SECONDS - 5 * 1440000;
```
Comment (thecadams) [inline src/components/organisms/ChatDrawer/ChatDrawer.tsx:66]: nit: would you be able to include some justification for the choice of constant here, as well as what 1440000 is in millis

**Review by thecadams — APPROVED** [review]
Proactively approving this one so you’re not blocked- but it’d be good to think about making this change not specific to Burning Seed.

**Comment by Demarchist1** [comment]
I moved the filter to the firebase call, hiding messages older than 2 weeks (constant in settings)

**Pull request merged**